website/docs/cloud-docs/users-teams-organizations/organizations.mdx

```diff
@@ -212,6 +212,24 @@ Organization owners can set up an SSO provider for the organization.
 
 #### General
 
```

Comment:
I do think we should briefly describe what a status check is, what it does, and why TFC uses it in this section.